src/py/flwr/server/strategy/dp_strategy_wrapper_fixed_clipping.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2020 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Central differential privacy with fixed clipping.
+
+Papers: https://arxiv.org/pdf/1712.07557.pdf, https://arxiv.org/pdf/1710.06963.pdf
+"""
+import warnings
+from typing import Dict, List, Optional, Tuple, Union
+
+from flwr.common import (
+    EvaluateIns,
+    EvaluateRes,
+    FitIns,
+    FitRes,
+    NDArrays,
+    Parameters,
+    Scalar,
+    ndarrays_to_parameters,
+    parameters_to_ndarrays,
+)
+from flwr.common.differential_privacy import (
+    add_gaussian_noise_to_params,
+    compute_clip_model_update,
+)
+from flwr.server.client_manager import ClientManager
+from flwr.server.client_proxy import ClientProxy
+from flwr.server.strategy.strategy import Strategy
+
+
+class DPStrategyWrapperServerSideFixedClipping(Strategy):
+    """Wrapper for Configuring a Strategy for Central DP with Server Side Fixed
+    Clipping.
+
+    Parameters
+    ----------
+    strategy: Strategy
```

Comment:
We're following the NumPy docstring style, which does the following:
```suggestion
    Parameters
    ----------
    strategy : Strategy
```